azure-functions/acbs-function/helpers/date.js

```diff
@@ -150,13 +152,7 @@ const getDateStringFromYearMonthDay = (year, month, day) => {
     return 'Invalid date';
   }
 
-  return formatOrReturnInvalidDate(
-    new Date(
-      Number(formatYear(year)),
-      Number(month) - 1,
-      Number(day),
-    ),
-  );
+  return formatOrReturnInvalidDate(new Date(Number(formatYear(year)), Number(month) - 1, Number(day)));
```

Comment:
Can the argument computation be moved to `variable` or a `function` to improve readbility?

Reply:
added -- sorry for rebase